src/dra/8A0A4.c

```diff
@@ -991,17 +991,25 @@ void func_8012CCE4(void) {
         // Might be possible to rewrite this block to reduce duplication with
         // some clever && and ||
         if (PLAYER.facingLeft) {
-            if ((g_Player.pl_vram_flag & 0xF000) == 0xC000) {
+            if ((g_Player.pl_vram_flag &
+                 (STANDING_ANY_SLOPE | STANDING_RAISING_SLOPE | VRAM_UNK2000 |
+                  STANDING_SLIGHT_SLOPE)) == 0xC000) {
                 PLAYER.velocityY = -(abs(PLAYER.velocityX) + FIX(3.5));
             }
-            if ((g_Player.pl_vram_flag & 0xF000) == 0x8000) {
```

Comment:
There's a bunch more of these so I'll stop pointing out every single one but it would make sense to document the compare values everywhere

Reply:
Yep, looks like the current linter will only handle one of masking or equality and not both. I've fixed these cases up manually